server/app.go

```diff
@@ -302,8 +299,7 @@ func startAmc(a *App, ctx context.Context) error {
 				return nil
 			}
 			for _, event := range eventMessage.Events {
-				// TODO: notify clients
-				go a.eventProcessor.Process(ctx, event)
+				a.eventProcessor.Process(ctx, event)
```

Comment:
in the same goroutine?

Reply:
yeap, because events should be processed sequentially

Reply:
sequentially in the scope of the current session